crates/algokit_http_client/src/lib.rs

```diff
@@ -54,7 +54,7 @@ pub struct HttpResponse {
 pub trait HttpClient: Send + Sync {
     async fn request(
         &self,
-        method: HttpMethod,
+        http_method: HttpMethod,
```

Comment:
See https://github.com/mozilla/uniffi-rs/issues/2628 for why this had to be renamed